resources/checkout-order-line-item-type.json

```diff
@@ -0,0 +1,70 @@
+{
+  "key": "checkout-order-line-item",
+  "name": {
+    "en": "Checkout order line item"
+  },
+  "description": {
+    "en": "Checkout order line item definition"
+  },
+  "resourceTypeIds": [
+    "line-item",
+    "custom-line-item"
+  ],
+  "fieldDefinitions": [
+    {
+      "name": "schedule",
+      "type": {
+        "name": "String"
+      },
+      "required": true,
+      "label": {
+        "en": "schedule"
+      },
+      "inputHint": "SingleLine"
+    },
+    {
+      "name": "isSubscription",
+      "type": {
+        "name": "Boolean"
+      },
+      "required": true,
+      "label": {
+        "en": "isSubscription"
+      },
+      "inputHint": "SingleLine"
+    },
+    {
+      "name": "subscriptionKey",
+      "type": {
+        "name": "String"
+      },
+      "required": true,
+      "label": {
+        "en": "subscriptionKey"
+      },
+      "inputHint": "SingleLine"
+    },
+    {
+      "name": "reminderDays",
+      "type": {
+        "name": "Number"
+      },
+      "required": false,
+      "label": {
+        "en": "reminderDays"
+      },
+      "inputHint": "SingleLine"
+    },
+    {
+      "name": "cutoffDays",
```

Comment:
cutoffDays might be cutOffDays

Reply:
- cut off is a verb

Reply:
Here is the difference (https://iconlogic.blogs.com/weblog/2008/08/grammar-workshop-one-word-or-two-confusing-compounds.html):
- cut off is a verb
- cutoff is used as a noun or adjective

In this case, the word cutoff is being used as adjective.